static/app/data/forms/userFeedback.tsx

```diff
@@ -18,6 +18,14 @@ const formGroups: JsonFormObject[] = [
         help: 'Show "powered by Sentry within the feedback dialog. We appreciate you helping get the word out about Sentry! <3',
         getData: data => ({options: data}),
       },
+      {
+        name: 'sentry:feedback_user_report_notifications',
+        type: 'boolean',
+
+        label: 'Enable Crash Report Notifications',
```

Comment:
Can we link to the docs section that explains what "crash report" is? I feel it's not that obvious for new Sentry users. Also perhaps "Modal" there somewhere helps folks realize what this is about

```suggestion
        label: 'Enable Crash Report Modal Notifications',
```